test/controllers/FinancialTransactionsControllerSpec.scala

```diff
@@ -16,23 +16,24 @@
 
 package controllers
 
-import utils.TestConstants.{fullFinancialTransactions,multipleDirectDebits}
+import utils.TestConstants.{fullFinancialTransactions, multipleDirectDebits}
 import base.SpecBase
 import controllers.actions.AuthActionImpl
 import mocks.auth.MockMicroserviceAuthorisedFunctions
 import mocks.services.MockFinancialTransactionsService
 import models._
 import play.api.http.Status
 import play.api.libs.json.Json
+import play.api.test.Helpers.{contentAsJson, defaultAwaitTimeout, status}
 
 class FinancialTransactionsControllerSpec extends SpecBase with MockFinancialTransactionsService with
   MockMicroserviceAuthorisedFunctions {
 
-  val singleError: Error = Error(code = "CODE", reason = "ERROR MESSAGE")
+  val singleError: Error = models.Error(code = "CODE", reason = "ERROR MESSAGE")
```

Comment:
Why was this package name syntax added in so many places? At least in this file, if I remove the package name it compiles and tests fine. If you don't have conflicting imports then you shouldn't need to state it.